pkg/processor/jsprocessor/builder.go

```diff
@@ -12,12 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package txfjs
+package jsProcessor
```

Comment:
I was thinking about `proc` too, as `processor` and `jsprocessor` are a bit long and "heavy". The reason why I left `processor` is because it describes a thing in the pipeline (similar to: source, destination, connector), and `proc` is used as an acronym for process. But that's not to say I don't like `procX`.: )